printmyblog.php

```diff
@@ -42,38 +43,39 @@ function pmb_already_active()
         <?php
     }
 
-    add_action('admin_notices', 'pmb_already_active', 1);
+    add_action('admin_notices', 'pmb_already_active', 1, 0);
```

Comment:
I suppose this is more correct (we don't expect any parameters to be passed in, whereas the default is actually to assume 1 is being passed).

Reply:
Yes. You're right.